doc/cug.tex

```diff
@@ -2780,87 +2780,26 @@ \subsubsection{Graph Types}
 \label{fig-test2}
 \end{figure}
 
-\subparagraph{Advanced Cycling}
-\label{AdvancedCycling}
-
-The hour-based section heading examples shown above (\lstinline=T00=,
-\lstinline=T06=, etc) are just one small part of cylc's cycling syntax.
+\subsubsection{Graph Headers}
 
-\lstinline=T06= stands for "Run every day starting at 06:00 after the
-initial cycle point". Cylc allows you to start (or end) at any particular
-time, repeat at whatever frequency you like, and even optionally limit the
-number of repetitions.
+In the above examples graph headings are used to define recurring date-times
+when tasks are to run. For example in the following scenario,
```

Comment:
I would say something like, "Graph section headings define the recurrence expressions (date-time or integer sequences, depending on the suite cycling mode) for their dependency graphs.".